samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/core/config/EventStoreDBConfig.java

```diff
@@ -0,0 +1,39 @@
+package io.eventdriven.ecommerce.core.config;
+
+import com.eventstore.dbclient.EventStoreDBClient;
+import com.eventstore.dbclient.EventStoreDBClientSettings;
+import com.eventstore.dbclient.EventStoreDBConnectionString;
+import com.eventstore.dbclient.ParseError;
+import io.eventdriven.ecommerce.api.backgroundworkers.EventStoreDBSubscriptionBackgroundWorker;
+import io.eventdriven.ecommerce.core.events.EventBus;
+import io.eventdriven.ecommerce.core.subscriptions.EventStoreDBSubscriptionCheckpointRepository;
+import io.eventdriven.ecommerce.core.subscriptions.SubscriptionCheckpointRepository;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class EventStoreDBConfig {
+
+  @Bean
+  @Scope("singleton")
+  EventStoreDBClient eventStoreDBClient() throws ParseError {
+    EventStoreDBClientSettings settings = EventStoreDBConnectionString.parse("esdb://localhost:2113?tls=false");
```

Comment:
Please define hardcoded value as "constant" in Java to ease maintenance later, or use configuration file if possible.
For example: 
``` Java
private final string esdbUrl = "esdb://localhost:2113?tls=false";
```

Reply:
I've added reading the connection string from `application.properties` in https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/150a618dd593b65939f9d758240afc066b545e4e.